src/lib/theme/type.tsx

```diff
@@ -3,12 +3,17 @@ import { Text, TextProps as TextPropsWithCss } from 'rebass'
 import styled, { useTheme } from './styled'
 import { Color } from './theme'
 
-type TextProps = Omit<TextPropsWithCss, 'css' | 'color'> & { color?: Color }
+type TextProps = Omit<TextPropsWithCss, 'css' | 'color' | 'userSelect'> & {
+  color?: Color
+  userSelect?: true
+}
 
-const TextWrapper = styled(Text)<{ color?: Color; lineHeight: string }>`
+const TextWrapper = styled(Text)<{ color?: Color; lineHeight: string; userSelect?: true }>`
   color: ${({ color = 'currentColor', theme }) => theme[color as Color]};
   // Avoid the need for placeholders by setting min-height to line-height.
   min-height: ${({ lineHeight }) => lineHeight};
+  // user-select must be configured through styled-components for cross-browser compat.
+  user-select: ${({ userSelect }) => userSelect && 'text'};
```

Comment:
- Can you add a comment here about `src\lib\components\Widget.tsx`'s `WidgetWrapper`'s `user-select: none`, explaining why this works the way it does? 

- I think the value should be `auto` here instead of `text`. [That should be more robust against edge case behavior.](https://developer.mozilla.org/en-US/docs/Web/CSS/user-select#syntax)

Reply:
- comment added
- `auto` will not work, because the parent has a value of `none`, so this will adopt a value of `none`

Reply:
`auto !important` or `unset`, then?

I'm surprised the simple `auto` doesn't work. I'd expected the value lower in the tree to take precedence. 🤷